util/network/src/host.rs

```diff
@@ -1150,7 +1150,7 @@ fn key_save_load() {
 #[test]
 fn host_client_url() {
 	let mut config = NetworkConfiguration::new();
-	let key = h256_from_hex("6f7b0d801bc7b5ce7bbd930b84fd0369b3eb25d09be58d64ba811091046f3aa2");
+	let key = "6f7b0d801bc7b5ce7bbd930b84fd0369b3eb25d09be58d64ba811091046f3aa2".into();
```

Comment:
why is this a simple `.into()` whereas the address stuff requires a `.parse().unwrap()`?